src/meta/src/manager/catalog/mod.rs

```diff
@@ -1743,6 +1744,95 @@ impl CatalogManager {
         Ok(version)
     }
 
+    pub async fn alter_owner(
+        &self,
+        entity: Entity,
+        owner_id: UserId,
+    ) -> MetaResult<NotificationVersion> {
+        let core = &mut *self.core.lock().await;
+        let database_core = &mut core.database;
+        let user_core = &mut core.user;
+
+        let notify_info;
+        match entity {
```

Comment:
```suggestion
        let notify_info = match entity {
```
Then you can return the info in each branch.

Reply:
As `commit_meta` moves out `table/schema/...`, if `notify_info` is returned by `match`, we need to pre-clone the `table/schema/...` before commit, which introduce one more line for each branch. Hence I think it's better to keep it original.